ansible/roles/ocp4-workload-dil-agile-integration/templates/user_registration.yaml.j2

```diff
@@ -0,0 +1,237 @@
+          livenessProbe:
+            failureThreshold: 3
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            successThreshold: 1
+            tcpSocket:
+              port: 6379
+            timeoutSeconds: 1
+          name: redis
+          ports:
+          - containerPort: 6379
+            protocol: TCP
+          readinessProbe:
+            exec:
+              command:
+              - "/bin/sh"
+              - "-i"
+              - "-c"
+              - "redis-cli -h 127.0.0.1 -a $REDIS_PASSWORD ping"
+            failureThreshold: 3
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            limits:
+              memory: 512Mi
+          securityContext:
+            capabilities: {}
+            privileged: false
+          terminationMessagePath: "/dev/termination-log"
+          terminationMessagePolicy: File
+          volumeMounts:
+          - mountPath: "/var/lib/redis/data"
+            name: redis-data
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        terminationGracePeriodSeconds: 30
+        volumes:
+        - name: redis-data
+          persistentVolumeClaim:
+            claimName: redis
+    test: false
+    triggers:
+    - type: ConfigChange
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    name: users
+    annotations:
+      app.openshift.io/connects-to: redis
+    labels:
+      app.kubernetes.io/part-of: registration
+      app.openshift.io/runtime: nodejs
+      app.kubernetes.io/name: app
+      app.kubernetes.io/component: server
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 10
+    selector:
+      app: users
+      deploymentconfig: users
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        creationTimestamp: 
+        labels:
+          app: users
+          deploymentconfig: users
+      spec:
+        containers:
+        - env:
+          - name: LAB_TITLE
+            value: '{{ ocp4_workload_dil_agile_integration_lab_title }}'
+          - name: LAB_BLOCKLIST
+            value: '1'
+          - name: LAB_USER_PREFIX
+            value: '{{ ocp4_workload_dil_agile_integration_username_prefix }}'
+          - name: LAB_USER_ACCESS_TOKEN
+            value: '{{ ocp4_workload_dil_agile_integration_user_access_token }}'
+          - name: LAB_USER_COUNT
+            value: '{{ num_supported_users }}'
+          - name: LAB_DURATION_HOURS
+            value: '2weeks'
+          - name: LAB_USER_PASSWORD
+            value: '{{ ocp4_workload_dil_agile_integration_openshift_user_password }}'
```

Comment:
For quoted strings like this it would be safer to use `to_json` to quote rather than `'`. For example:
```
            value: {{ ocp4_workload_dil_agile_integration_openshift_user_password | to_json }}
```